core/views.py

```
@@ -181,46 +195,286 @@ class TermsOfServiceView(TemplateView):
     template_name = "pages/terms_of_service.html"
 
 
-def create_checkout_session(request, pk, plan):
+# views.py
+
+
+@login_required
+def create_checkout_session(request, pk, product_name):
```

Ruff (0.14.2)

Unused function argument: `pk`

(ARG001)